### PR TITLE
Add documentation howto build in docker and create a custom docker image

### LINKED ISSUE #9387: Document how to build docker image
It would be helpful if there where some more details in [/docs/sources/project/building_from_source.md](https://github.com/grafana/grafana/blob/master/docs/sources/project/building_from_source.md) on how to build your own docker image from master/PR etc.

### PR DESCRIPTION
**Pull request by rickard-von-essen**
Add documentation on how to:

- Build Linux packages in Docker
- Build a custom Docker image

Closes #9387

### CODE REVIEW

**Comment by torkelo** [comment]
Thanks for wanting to contribute and wanting to make grafana docs better! 

not 100% sure this belongs in this docs article, maybe a separate guide or blog post?

**Comment by torkelo** [comment]
thoughts @bergquist @daniellee ?

**Comment by rickard-von-essen** [comment]
I think it belongs in the docs, maybe in another (new article) but putting it in a blog post would make it hard to find for the ones who are contributing.

**Comment by bergquist** [comment]
I think this belong in the docs. Either a build instruction or as an tutorial.  

I think we should splitt the build from source into three different pages
* Building from source
* How to get started developing Grafana
* Building custom docker containers 

Does that make sense?

**Comment by daniellee** [comment]
I agree with @bergquist It would be a nice addition to the docs but should be split out from the Building from source page.

**Comment by rickard-von-essen** [comment]
Will you guys take on this or should I make a draft? If you'd like me to do it it would be great if have some pointers, or should I just go with @bergquist's https://github.com/grafana/grafana/pull/9421#issuecomment-334397638?

File: docs/sources/project/building_from_source.md [hunk 1]
```diff
@@ -92,6 +92,59 @@ This step builds linux packages and requires that fpm is installed. Install fpm
...
+
+```
+docker run -itv $PWD:/go/src/github.com/grafana/grafana -w /go/src/github.com/grafana/grafana ubuntu:xenial bash -c '
+apt-get update &&
+apt-get install -y software-properties-common &&
+add-apt-repository -y ppa:gophers/archive &&
+apt-get install -y curl &&
+curl -sL https://deb.nodesource.com/setup_6.x | bash - &&
+apt-get update &&
+apt-get install -y rpm git golang-1.9-go nodejs ruby ruby-dev &&
```
Comment (rickard-von-essen-iz) [inline docs/sources/project/building_from_source.md]: `fontconfig` is missing

**Comment by bergquist** [comment]
@rickard-von-essen sorry for loosing this. If you want to take a stab at writing the docs it would be **very** appreciated. Assigning this to myself to make sure we don't loose track again.

**Comment by bergquist** [comment]
Closing this PR and opened an issue for what we discussed. https://github.com/grafana/grafana/issues/10384

**Pull request closed without merging**